templates/document_categories/general_list.html

```diff
@@ -15,25 +15,43 @@
 {/literal}
 <script type="text/javascript">
 var deleteLabel="{xl t="Delete"}";
+var editLabel="{xl t="Edit"}";
 </script>
-<script type="text/javascript" src="{$WEBROOT}/library/js/CategoryTreeMenu.js"></script>
+<script type="text/javascript" src="{$WEBROOT}/library/js/CategoryTreeMenu.js?v=3"></script>
```

Comment:
does it make sense to instead utilize the $v_js_includes and then increment that?